tests/unit/acl.tcl

```diff
@@ -465,6 +465,33 @@ start_server {tags {"acl external:skip"}} {
         }
     }
 
+    test "ACL CAT with illegal arguments" {
+        assert_error {*Unknown category 'NON_EXISTS'} {r ACL CAT NON_EXISTS}
+        assert_error {*Unknown subcommand or wrong number of arguments for 'CAT'*} {r ACL CAT NON_EXISTS NON_EXISTS2}
+    }
+
+    test "ACL CAT without category - list all categories" {
+        set reply [r acl cat]
+        assert_match "*keyspace*" $reply
+        assert_match "*connection*" $reply
+    }
+
+    test "ACL CAT category - list all commands that belong to category" {
+        # We simply just check a few categories.
+        assert_match "*llen*" [r ACL CAT list]
+        assert_match "*zcard*" [r ACL CAT sortedset]
+        assert_match "*exists*" [r ACL CAT keyspace]
+        assert_match "*flushall*" [r ACL CAT dangerous]
+    }
```

Comment:
why do we need this test? isn't the test below sufficient?
i.e. we're testing the infrastructure, not all possible commands and categories.